app/code/community/Reverb/ReverbSync/Model/Sync/Order/Update.php

```diff
@@ -61,7 +67,8 @@ protected function _executeStatusUpdate($magento_order_entity_id, $reverb_order_
             // Fire event to allow for executing functionality upon order cancels/refunds
             Mage::dispatchEvent($event_name,
                                     array('order_entity_id' => $magento_order_entity_id,
-                                          'reverb_order_status' => $reverb_order_status)
+                                          'reverb_order_status' => $reverb_order_status,
+                                          'reverb_order_update_arguments_object' => $argumentsObject)
             );
```

Comment:
Ok. This should handle 99% of the cases as an order usually gets to those states. (even though there are other states that happen in the middle).